### ISSUE TITLE
It seems that intellij is not properly referencing the type.

### ISSUE THREAD

**Issue by MIN0707** [roblox-ts/types#946]
`import {Players} from "@rbxts/services";

const player:Player = Players.LocalPlayer

print(player.Name)`

In this code, autocomplete is not properly supported after the player.
because the red line
It says unresolvable variable
But the code works fine.

My guess is that the references in the PluginSecurity.d.ts and None.d.ts files don't seem right.

**Comment by LouieK22** [comment]
On the latest version of Webstorm (2023.1.2), the reference is resolving correctly and `Player.Name` is interpreted as a `string`. However, all references to `Instances` are broken (e.g. `t.instanceOf()`) since `Instance` gained the `AddTag` method which is invalid when inherited by `CollectionService`, producing this error:
```
TS2430: Interface 'CollectionService' incorrectly extends interface 'Instance'.
Types of property 'AddTag' are incompatible.
Type '(this: CollectionService, instance: Instance, tag: string) => void' is not assignable to type '(this: Instance, tag: string) => void'.
```

**Comment by Dionysusnu** [comment]
> [] all references to `Instances` are broken (e.g. `t.instanceOf()`) since `Instance` gained the `AddTag` method which is invalid when inherited by `CollectionService`, producing this error:
> 
> ```
> TS2430: Interface 'CollectionService' incorrectly extends interface 'Instance'.
> Types of property 'AddTag' are incompatible.
> Type '(this: CollectionService, instance: Instance, tag: string) => void' is not assignable to type '(this: Instance, tag: string) => void'.
> ```

For reference: This was fixed in #964
